django/contrib/gis/db/backends/base/features.py

```diff
@@ -46,6 +46,9 @@ class BaseSpatialFeatures:
     # Does the database support a unique index on geometry fields?
     supports_geometry_field_unique_index = True
 
+    # Can the backend GROUP BY a geometry?
+    can_group_by_geometry = True
```

Comment:
I'm not sure exactly what Oracle's limitation is. This feature might not be correctly named. Here are the failing queries from the three tests:
```sql
SELECT "GEOAPP_CITY"."ID", SDO_UTIL.TO_WKBGEOMETRY("GEOAPP_CITY"."POINT"), COUNT("GEOAPP_CITY"."POINT") AS "POINT__COUNT" FROM "GEOAPP_CITY" GROUP BY "GEOAPP_CITY"."ID", SDO_UTIL.TO_WKBGEOMETRY("GEOAPP_CITY"."POINT"); 


SELECT "RELATEDAPP_LOCATION"."ID", SDO_UTIL.TO_WKBGEOMETRY("RELATEDAPP_LOCATION"."POINT"), COUNT("RELATEDAPP_CITY"."ID") AS "NUM_CITIES" FROM "RELATEDAPP_LOCATION" LEFT OUTER JOIN "RELATEDAPP_CITY" ON ("RELATEDAPP_LOCATION"."ID" = "RELATEDAPP_CITY"."LOCATION_ID") WHERE "RELATEDAPP_LOCATION"."ID" = 5 GROUP BY "RELATEDAPP_LOCATION"."ID", SDO_UTIL.TO_WKBGEOMETRY("RELATEDAPP_LOCATION"."POINT") FETCH FIRST 21 ROWS ONLY;


SELECT "RELATEDAPP_LOCATION"."ID", SDO_UTIL.TO_WKBGEOMETRY("RELATEDAPP_LOCATION"."POINT"), COUNT("RELATEDAPP_CITY"."ID") AS "NUM_CITIES" FROM "RELATEDAPP_LOCATION" LEFT OUTER JOIN "RELATEDAPP_CITY" ON ("RELATEDAPP_LOCATION"."ID" = "RELATEDAPP_CITY"."LOCATION_ID") WHERE "RELATEDAPP_LOCATION"."ID" = 5 GROUP BY "RELATEDAPP_LOCATION"."ID", SDO_UTIL.TO_WKBGEOMETRY("RELATEDAPP_LOCATION"."POINT");
```

Reply:
You cannot group by LOBs on Oracle, see ticket-24096.

Reply:
The description of [SDO_UTIL.TO_WKBGEOMETRY](https://docs.oracle.com/database/121/SPATL/sdo_util-to_wkbgeometry.htm#SPATL1250) is "Converts a Spatial and Graph geometry object to the well-known binary (WKB) format." So perhaps the feature name could be `can_group_by_binary`, which wouldn't be specific to the GIS backend and might be useful for other tests.

Reply:
Sounds good :+1: